servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HttpExecutionContextBuilder.java

```diff
@@ -71,4 +72,12 @@ public HttpExecutionContext build() {
                 executor == null ? defaultContextSupplier.get().executor() : executor,
                 strategy == null ? defaultStrategy() : strategy);
     }
+
+    static <U, R> SingleAddressHttpClientBuilder<U, R> setExecutionContext(
```

Comment:
It seems strange to have this as a method on `HttpExecutionContextBuilder` when none of the types involved is this type. Can we somehow put this on `SingleAddressHttpClientBuilder`?